README.rst

```diff
@@ -56,4 +78,4 @@ follow the instructions in ``etstool.py``.  In particular::
 
     > python etstool.py test-all
 
-will run tests in all supported environments automatically.
\ No newline at end of file
+will run tests in all supported environments automatically.
```

Comment:
Ah... I tried to change 'wxPython' to 'wxPython4' via GitHub directly and GitHub did this change for me!

Reply:
This adds the missing newline at the end of file, and I am going to merge away...